ts/components/staking/wizard/wizard_flow.tsx

```diff
@@ -312,20 +313,25 @@ export const WizardFlow: React.FC<WizardFlowProps> = props => {
                 topLabels={['Amount', `Available: ${formattedAmount} ZRX`]}
                 labels={[StakingPercentageValue.Fourth, StakingPercentageValue.Half, StakingPercentageValue.All]}
                 value={stakeAmount}
+                selectedLabel={selectedLabel}
```

Comment:
Ah didn't realize this was a thing. When you type does it reset to no selected label value?

Reply:
Yeah, exactly.